tests/test_nginx.py

```diff
@@ -56,3 +56,22 @@ def test_regex():
         n.clear_attributes()
         n.parse(log)
         assert message == n.message
+
+
+def test_error_does_not_match_regex():
+    """
+    Test when the regex does not match
+    """
+    logs = (
+        'nginx: [emerg] open() "/etc/nginx/mime.types" failed (2: No such file or directory) in /opt/bitnami/nginx/conf/server_blocks/server-block.conf:24',  # noqa: E501
```

Comment:
1. Please, split up this line. There should be no need for `# noqa`.
1. While a tuple works a list would be more appropriate here. (Read: Please use square brackets.)

Reply:
1. Why split up the line? It is a test string and I think it reads better as one line.
2. Why not use a tuple? The list is not going to be modified.

Reply:
"Beautiful is better than ugly."

ad 1. `# noqa: E501` should be removed.
ad 2. This is semantically a list. Hence, a list would be more appropriate.

Reply:
I have made the changes even though I think one line makes it more beautiful and that it should be a tuple.

You should change the rest of the file to match what you want.